.gitignore

```diff
@@ -41,4 +41,6 @@ Make.dep
 .DS_Store
 build
 .ccls-cache
-compile_commands.json
\ No newline at end of file
+compile_commands.json
+# keep dlang source files
+!examples/hello_d/*.d
```

Comment:
should we remove line 4?

Reply:
@kassane `.ignore` has this at line 4. Perhaps we should remove this, so we don't need `!examples/hello_d/*.d`? 

https://github.com/apache/nuttx-apps/blob/1f443afcc95a87114c226747e7aa90df17f8d05c/.gitignore#L4

(I wonder why we are skipping `.d` files in the first place?)

Reply:
done!